generators/app/index.js

```diff
@@ -95,6 +97,34 @@ module.exports = generators.Base.extend({
       }.bind(this));
     },
 
+    askForGithubAccount: function () {
+      var done = this.async();
+
+      githubUsername(this.user.git.email(), function (err, username) {
+        if (err) {
+          this.log.error(err);
+        }
+
+        this.prompt({
+          name: 'githubAccount',
+          message: 'GitHub username or organization',
+          default: username
+        }, function (prompt) {
+          if (prompt.githubAccount) {
+            this.props.githubAccount = prompt.githubAccount;
+            originUrl(this.destinationPath(), function (err, url) {
```

Comment:
Shouldn't we check this before hands?

What is the use of the githubAccount name if we don't need to setup git?

---

If we correctly need to run this after, then we should move this inside of `node:git` because it is not used elsewhere.